src/unistd.rs

```diff
@@ -631,6 +631,8 @@ pub fn chown<P: ?Sized + NixPath>(path: &P, owner: Option<Uid>, group: Option<Gi
 pub enum FchownatFlags {
     FollowSymlink,
     NoFollowSymlink,
+    #[cfg(target_os = "linux")]
+    EmptyPath,    
```

Comment:
If we add this flag, then this type should no longer be an enum but a bitflag cause you can do `AT_EMPTY_PATH` and ` AT_SYMLINK_NOFOLLOW` at the same time